app/routes/products_.$productId/route.tsx

```diff
@@ -66,9 +66,28 @@ export default function ProductDetailsPage() {
                 className={styles.left}
             />
             <div className={styles.right}>
-                <div>{product.name}</div>
+                <div>
+                    <div className={styles.productName}>{product.name}</div>
+                    {product.sku !== undefined && (
+                        <div className={styles.sku}>SKU: {product.sku}</div>
+                    )}
+                </div>
+
                 {product.priceData && (
-                    <div className={commonStyles.price}>{product.price?.formatted?.price}</div>
+                    <div className={styles.priceSection}>
+                        {product.priceData?.price !== product.priceData?.discountedPrice && (
```

Comment:
are you required to add `?.` after priceData, taking into account that you have this check `{product.priceData &&` ?

Reply:
you are right, removed unneeded `?`s